docs/en/operation/Troubleshooting.md

```diff
@@ -204,19 +207,40 @@ or [other approaches](https://stackoverflow.com/questions/880557/socket-accept-t
 value in `conf/alluxio-site.properties`. Note that, reducing the value of these two properties may
 potentially add latency for master to serve requests.
 
-TODO(feng) add a section about discover file updates in under fs.
+### Q: I added some files in under file system. How can I reveal the files in Alluxio?
+
+By default, Alluxio loads the list of files the first time a directory is visited. Alluxio will keep using the cached file list regardless of the changes in the under file system. To reveal new files from under file system, you can use the command `alluxio fs ls -f /some/path` to manually discover the new content inside a specific folder. Another way to refresh a directory is to use UFS sync.
+You can either use it in command line by running `alluxio fs ls -R -Dalluxio.user.file.metadata.sync.interval=${SOME_INTERVAL} /path` or by setting the same configuration property in masters' `alluxio-site.properties`. The value for the configuration property is used to
+determine the minimum interval between two syncs. You can read more about loading files from under file system [here](../advanced/Loading-Under-File-Storage-Metadata.html).
+
+
+### Q: I see an error "Block ?????? is unavailable in both Alluxio and UFS" while reading some file. Where is my file?
```

Comment:
this error is also shown when the block exists in the UFS, but there are no workers available (e.g. all workers have died)

Reply:
Added a section to explain the alternative cause.